src/main/java/edu/hm/hafner/analysis/parser/Gcc4CompilerParser.java

```diff
@@ -143,20 +148,28 @@ private String getNotes(final LookaheadStream lookahead) {
     }
 
     @SuppressWarnings("PMD.AvoidLiteralsInIfCondition")
-    static boolean isMessageContinuation(final LookaheadStream lookahead) {
+    static boolean isMessageContinuation(final LookaheadStream lookahead, final boolean hasCodeSnippet) {
         var peek = lookahead.peekNext();
         if (peek.length() < 3) {
```

Comment:
Move the if into the extracted method, it does not make sense as a standalone check.